pom.xml

```diff
@@ -73,6 +71,117 @@
 		</repository>		
 	</repositories>
 
+	<dependencyManagement>
```

Comment:
Is there a reason for moving these dependencies?  My initial thought on leaving them was that they weren't necessarily needed by every sub-project and shouldn't be included if not.

Reply:
This is the `<dependencyManagement>` block, which is effectively the "BOM" of the project.  Corey set up these versions to line up with the CQL engine and its dependencies back when `cohort-engine` was the only project.  These shouldn't force any dependencies on any subprojects, just the versions.

Reply:
It feels a little weird to call out the versions specifically above *and* have the dependencyManagement block... but I guess I don't care that much.

Reply:
Kwas explained that the versions are used for multiple dependencies, so I take it back.

Reply:
Yep. This is very important. I already ran into a case where cql-ruler was pulling in a different version of the engine than we were intending to use. Dependency management smooths it all out.

Reply:
Thanks for the discussion. Had a similar train of thought as Jill, but this cleared things up.